bot/exts/moderation/incidents.py

```diff
@@ -130,6 +138,39 @@ def has_signals(message: discord.Message) -> bool:
     return ALL_SIGNALS.issubset(own_reactions(message))
 
 
+async def make_message_link_embed(ctx: Context, message_link: str) -> discord.Embed:
+    """
+    Create an embed representation of discord message link contained in the incident report.
+
+    The Embed would contain the following information -->
+        Author: @Jason Terror ♦ (736234578745884682)
+        Channel: Special/#bot-commands (814190307980607493)
+        Content: This is a very important message!
+    """
+    embed = discord.Embed()
+
+    try:
+        message_convert_object = MessageConverter()
+        message = await message_convert_object.convert(ctx, message_link)
+
+    except Exception as e:
```

Comment:
This shouldn't be a bare exception

Reply:
In many places of the code we use a bare exception, and for the similar case I using this, If any error occurs it should be mentioned in the embed, whether it is Channel Not Found, Channel Unreadable or Message Not Found. This is then set as the header of the error embed for that particular message link.

Reply:
Maybe `DiscordException`?

Reply:
Yeah, `DiscordException` would be better.